ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -130,6 +133,25 @@
       (.setHost (options :host))
       (.setIdleTimeout (options :max-idle-time 200000)))))
 
+(defn- socket-connector
+  ^UnixSocketConnector [^Server server {:keys [unix-socket] :as options}]
+  (when (->> (System/getProperty "os.name")
+             (re-find #"(?i)^windows"))
+    (throw (ex-info "Unix sockets not supported on windows"
+                    {:os (System/getProperty "os.name")
+                     :unix-socket unix-socket})))
+  (let [http-factory (HttpConnectionFactory. (http-config options))
+        socket (io/file unix-socket)]
+    (when (.exists socket)
+      (throw (ex-info "File already exists at socket path; should be deleted before starting server"
+                      {:unix-socket unix-socket})))
+    (.deleteOnExit socket)
+    (doto (UnixSocketConnector.
+            server
+            #^"[Lorg.eclipse.jetty.server.ConnectionFactory;" (into-array ConnectionFactory [http-factory]))
```

Comment:
Can you split the line over 80 characters into two lines? Also can you fix the indentation so it's according to the style guide:

```clojure
(doto (UnixSocketConnector.
       server
       ^"[Lorg.eclipse.jetty.server.ConnectionFactory;"
       (into-array ConnectionFactory [http-factory]))
```